pkg/controllers/machinesetsync/machineset_sync_controller_test.go

```
@@ -663,6 +663,47 @@ var _ = Describe("With a running MachineSetSync controller", func() {
+				Eventually(k.Update(mapiMachineSet, func() {
+					mapiMachineSet.Spec.AuthoritativeAPI = mapiv1beta1.MachineAuthorityClusterAPI
+					// spec.template.spec.authoritativeAPI stays as MachineAPI (the API server default),
+					// simulating a stub created with only spec.authoritativeAPI: ClusterAPI set.
+					mapiMachineSet.Spec.Template.Spec.AuthoritativeAPI = mapiv1beta1.MachineAuthorityMachineAPI
+				})).Should(Succeed())
+
+				By("Creating the CAPI infra machine template")
+				Eventually(kCreate(ctx, capaMachineTemplate)).Should(Succeed(), "capa machine template should be able to be created")
+
+				By("Creating the CAPI machine set")
+				capiMachineSet = capiMachineSetBuilder.Build()
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)